src/elexmodel/models/BootstrapElectionModel.py

```diff
@@ -806,36 +835,27 @@ def compute_bootstrap_errors(
             nonreporting_units, "turnout_factor"
         )
 
-        # we use k-fold cross validation to find the optimal lambda for our OLS regression
-        optimal_lambda_y = self.cv_lambda(x_train, y_train, np.logspace(-3, 2, 20), weights=weights_train)
-        optimal_lambda_z = self.cv_lambda(x_train, z_train, np.logspace(-3, 2, 20), weights=weights_train)
-
         # step 1) fit the initial model
         # we don't want to regularize the intercept or the coefficient for baseline_normalized_margin
-        ols_y = OLSRegressionSolver()
-        ols_y.fit(
-            x_train,
-            y_train,
-            weights=weights_train,
-            lambda_=optimal_lambda_y,
-            fit_intercept=True,
-            regularize_intercept=False,
-            n_feat_ignore_reg=1,
-        )
-        ols_z = OLSRegressionSolver()
-        ols_z.fit(
-            x_train,
-            z_train,
-            weights=weights_train,
-            lambda_=optimal_lambda_z,
-            fit_intercept=True,
-            regularize_intercept=False,
-            n_feat_ignore_reg=1,
-        )
+        model_kwargs = {"fit_intercept": True, "regularize_intercept": False, "n_feat_ignore_reg": 1}
+        if self.model_type == "OLS":
+            # if we run OLS we use cross validation to find the optimal lambda
+            # TODO: why do we not do this for other models also
```

Comment:
lol wait, why do we not do this for other models also? 🤔 

Is this a "how can I write code to make this happen" question or a theoretical question (or both)? 🤔